### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by interfector**
These are 4 separate commits for various fixes.

### CODE REVIEW

File: Magi/Sofia of Criamon/Sofia, the Dream Witch+15.md [hunk 1]
```diff
@@ -109,7 +109,7 @@
 + *Posing the Silent Question* (InMe 20/+20)
 + *Lay to Rest the Haunting Spirit* (PeMe 10/+18)
 + *Demon's Eternal Oblivion* (PeVi 10/+13)
-+ *Opening the Portal to the Dreamscape* (Mu(In)Me 30/+27)
++ *Opening the Portal to the Dreamscape*<sup>F</sup> (Mu(In)Me 30/+27)
```
Comment (GuillaumeDIDIER) [inline Magi/Sofia of Criamon/Sofia, the Dream Witch+15.md:112]: This is missing a legend as to the meaning of the F (in superscript)

File: Magi/Sofia of Criamon/Sofia, the Dream Witch+30.md [hunk 2]
```diff
@@ -111,16 +111,16 @@
 + *Posing the Silent Question* (InMe 20/+24)
 + *Lay to Rest the Haunting Spirit* (PeMe 10/+24)
 + *Demon's Eternal Oblivion* (PeVi 10/+17)
-+ *Opening the Portal to the Dreamscape* (Mu(In)Me 30/+32)
-+ *Blissful Dreams* (ReMe 20/+34)
++ *Opening the Portal to the Dreamscape*<sup>F</sup> (Mu(In)Me 30/+32)
++ *Blissful Dreams*<sup>F</sup> (ReMe 20/+34)
```
Comment (GuillaumeDIDIER) [inline Magi/Sofia of Criamon/Sofia, the Dream Witch+30.md:115]: Same as above what does <sup>F</sup> mean ?

**Review by GuillaumeDIDIER — CHANGES_REQUESTED** [review]

**Comment by GuillaumeDIDIER** [comment]
In practice rebase -i and edit the third commit should do the trick to rewrite the offending commit.

**Comment by interfector** [comment]
I should have fixed the issue and commit another fix to the magic items.

**Comment by interfector** [comment]
Sorry for the requested review, I was trying different things to *solve* this **Changes requested** 😟

**Review by GuillaumeDIDIER — APPROVED** [review]

**Pull request merged**